editor-extensions/vscode/server/src/server.ts

```diff
@@ -48,6 +48,9 @@ function filenameFromUri(textDocumentUri: string) {
     if (filename.startsWith("/")) {
       filename = filename.substring(1);
     }
+
+    // Packaged Grain doesn't understand lowercase drive letters
+    filename = filename[0].toUpperCase() + filename.substring(1);
   }
```

Comment:
This is OK, but  do we always have the drive letter as the first letter?  If not, and in the unusual case of someone enabling case sensitive file system on Windows 10, this could break.

Reply:
No reason to hold this update for that though!